pkg/apiserver/registry/stats/nodelatencystats/rest.go

```diff
@@ -50,38 +57,80 @@ func (r *REST) Destroy() {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	summary := obj.(*statsv1alpha1.NodeLatencyStats)
+	err := r.indexer.Update(summary)
+	if err != nil {
+		return nil, errors.NewInternalError(err)
+	}
+
+	return summary, nil
 }
 
 func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	obj, exists, err := r.indexer.GetByKey(name)
+	if err != nil || !exists {
+		return nil, errors.NewNotFound(statsv1alpha1.Resource("nodelatencystats"), name)
+	}
```

Comment:
it probably doesn't matter here as I don't think this function can actually return an error, but I think you should handle both cases differently: use `NewInternalError` for an error, and `NewNotFound` if there is no error and `exists` is false.

Reply:
Ok, I will split these conditions.